packages/web/src/scss/components/UNSTABLE_Slider/index.html

```diff
@@ -21,13 +21,12 @@ <h2 class="docs-Heading">Default</h2>
       <input
         class="UNSTABLE_Slider__input"
         id="slider-steps"
-        style="--slider-position: 30%"
+        style="--slider-position: calc(100% * (9 - 3) / (12 - 3))"
         type="range"
-        min="0"
-        max="10"
-        step="1"
-        value="3"
-        oninput="this.style.setProperty('--slider-position', `${Math.round((100 * this.value) / 10)}%`);"
+        min="3"
+        max="12"
+        value="9"
+        oninput="this.style.setProperty('--slider-position', `${Math.round(100 * (this.value - 3) / (12 - 3))}%`);"
```

Comment:
Name `Custom Steps` evoked in me that the demonstration will include settings of `max`, `min`, `step` and will display the currently selected value (so a small playground like in the case of Toast, etc.), but I could imagine that the time wasn't on your side and we can extend this demo later.

Reply:
Btw, couldn't this be part of CSS?

```
.UNSTABLE_Slider__input {
    --slider-position: math.round(100 * (attr(value) - attr(min)) / (attr(max) - (attr(min))));`
    …
}
```
?